docs/source/optimize.rst

```diff
@@ -106,6 +106,7 @@ tasks to improve efficiency using the ``inline`` function. For example:
 .. code-block:: python
 
     >>> from dask.optimize import inline
+    >>> dependencies = {k: set(deps) for k, deps in dependencies.items()}
```

Comment:
Fixed

On Thu, Sep 8, 2016 at 12:31 PM, Jim Crist <notifications@github.com> wrote:

> In docs/source/optimize.rst
> <https://github.com/dask/dask/pull/1531#discussion_r78041368>:
>
> > @@ -106,6 +106,7 @@ tasks to improve efficiency using the ``inline`` function. For example:
> >  .. code-block:: python
> >
> >      >>> from dask.optimize import inline
> > +    >>> dependencies = {k: set(deps) for k, deps in dependencies.items()}
>
> Perhaps just add a check in inline instead?
>
> if dependencies is None:
>     ...elif dependencies and isinstance(dependencies.values()[0], list):
>     dependencies = {k: set(v) for k, v in dependencies.items()}
>
> Either is fine by me though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dask/dask/pull/1531/files/1b7d94c6f1fe9252390f77787eeb9e0c3848eeb1#r78041368>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASszA7GZy75Fpl2y5ylrhWxw_DFFR03ks5qoDhZgaJpZM4J4Klh>
> .
>